.gitignore

```diff
@@ -1 +1,6 @@
-node_modules
\ No newline at end of file
+<<<<<<< HEAD
+node_modules
+test2.js
+=======
+node_modules
+>>>>>>> a3db5f3caa32708acf0737761ad73d07fd56a26d
```

Comment:
CRITICAL: Merge conflicts should not find it's way to your codebase. Ensure they are fixed locally before you push.

NITPICK: I'm sure there are more files than these you would want to ignore e.g. .DS_Store